introduction.md

```diff
@@ -3,4 +3,11 @@ Introduction
 
 I'm a Copenhagen based software engineer and I'm entering this dojo with an open mind.
 
-Favorite CodeSchool path: Ruby 
+Favorite CodeSchool path: Ruby
+
+
+Deadly Skills
+-------------
+* Git
+* Ruby
+* JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?